cmd/hpsf/main.go

```diff
@@ -143,12 +143,21 @@ func main() {
 		}
 
 		// validate the HPSF
-		errors := hpsf.Validate()
-		if len(errors) > 0 {
-			for _, e := range errors {
-				log.Printf("error: %v", e)
+		verrors := hpsf.Validate()
+		if verrors == nil {
+			log.Printf("HPSF is valid")
+			os.Exit(0)
+		}
+
+		if hErr, ok := verrors.(validator.Error); ok {
```

Comment:
The valid HPSF log entry after the if/else is now redundant. Can we move the os.Exit to after the if/else or change the verrors == nil to != and use the os.Exit inside that branch?